configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -219,13 +218,22 @@
         </p>
     </div>
 
+
+
     <ifMode mode="VIEW" include="false">
         <h2>
             <label>
                 <uimessage code="pihcore.mentalhealth.liberia.title"/>
             </label>
         </h2>
 
+        <span id="mh_mrn_enter" style="float:right;">
+            <label class="inline">MRN:</label>
+            <span class="inline">
+                <patient field="identifier" identifierTypeId="23507d1e-4f22-11ea-9717-645d86728797" required="false"/>
```

Comment:
Just as a heads up, we haven't used the "patient" HTML Form Entry tag in a long time (except for possibly in Rwanda) so would be worth a little testing (which you may have done already) just to make sure it works probably with OpenMRS 2.x

Reply:
Yes it works but I had thought of removing it altogether because we now have the identifier editable on patient header.